package.json

```diff
@@ -13,6 +13,8 @@
     "precommit": "lint-staged",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "commit": "git-cz",
+    "build:changelog": "lerna-changelog --from=$(git log --grep='chore(release): releasing component' --format='%H' | sed -n 2p) > tmp && cat CHANGELOG.md >> tmp && cat tmp > CHANGELOG.md && rm tmp",
+    "push:changelog": "git add CHANGELOG.md && git commit -m 'docs: update changelog' && git push",
```

Comment:
then we git add the changelog, commit it and push it.

Reply:
Could also just be `post:build:changelog`?

Reply:
Putting them all into one script?